docs/snippets/angular/storybook-preview-global-decorator.ts.mdx

```diff
@@ -0,0 +1,16 @@
+```js
+// .storybook/preview.js
+
+export const decorators = [
+  (storyFunc) => {
+    const story = storyFunc();
+
+    return {
+      ...story,
+      template: `<div style="margin: 3em">${story.template}</div>`,
+    };
+  },
+];
+
+// Note: A template with the component's selector is required in each component story for the decorators to take place.
```

Comment:
@profanis if you're ok, move the comment above and place it above the `decorators` and we could possibly reword it as follows 

" To see the global decorator applied to your stories you'll need to include the template key with the component's selector with each story you have."

Reply:
I like your suggestion 😊.

I updated it accordingly